docs/docs-content/user-management/palette-rbac/create-custom-role.md

```diff
@@ -0,0 +1,218 @@
+
+5. Provide a name for the custom role in the **Role Name** field.
+
+6. Select the permissions you want to assign to the custom role.
+
+7. Click on the **Save** button.
+
+### Validate
+
+To validate that the custom Project role is created successfully, follow these steps:
+
+1. Log in to [Palette](https://console.spectrocloud.com) as a tenant admin.
+
+2. Navigate to the left **Main Menu** and select **Roles**.
+
+3. Navigate to the **Project Roles** tab.
+
+4. Verify that the custom role is listed in the **Project Roles** tab.
+
+5. Assign the custom Project role to a user or group to validate that the permissions are applied correctly.
+
+## Resource Roles
+
+To create a custom resource role in Palette, use the following steps.
+
+### Prerequisites
+
+- You need Tenant admin permissions to create a custom Resource role.
+
+### Create a Custom Resource Role
+
+1. Log in to [Palette](https://console.spectrocloud.com) as a tenant admin.
+
+2. Navigate to the left **Main Menu** and select **Roles**.
+
+3. Navigate to the **Resource Roles** tab.
+
+4. Click on the **Create Resource Role** button at the top right corner of the page.
+
+5. Provide a name for the custom role in the **Role Name** field.
+
+6. Select the permissions you want to assign to the custom role.
+
+7. Click on the **Save** button.
+
+### Validate
+
+To validate that the custom Resource role is created successfully, follow these steps:
+
+1. Log in to [Palette](https://console.spectrocloud.com) as a tenant admin.
+
+2. Navigate to the left **Main Menu** and select **Roles**.
+
+3. Navigate to the **Project Roles** tab.
+
+4. Verify that the custom role is listed in the **Project Roles** tab.
+
+5. Assign the custom Resource role to a user or group to validate that the permissions are applied correctly.
+
+## Tenant Roles
+
+To create a custom tenant role in Palette, use the following steps.
+
+### Prerequisites
+
+- You need Tenant admin permissions to create a custom Tenant role.
+
+### Create a Custom Tenant Role
+
+1. Log in to [Palette](https://console.spectrocloud.com) as a tenant admin.
+
+2. Navigate to the left **Main Menu** and select **Roles**.
+
+3. Navigate to the **Tenant Roles** tab.
+
+4. Click on the **Create Tenant Role** button at the top right corner of the page.
+
+5. Provide a name for the custom role in the **Role Name** field.
+
+6. Select the permissions you want to assign to the custom role.
+
+7. Click on the **Save** button.
+
+### Validate
+
+To validate that the custom Tenant role is created successfully, follow these steps:
+
+1. Log in to [Palette](https://console.spectrocloud.com) as a tenant admin.
+
+2. Navigate to the left **Main Menu** and select **Roles**.
+
+3. Navigate to the **Project Roles** tab.
+
+4. Verify that the custom role is listed in the **Project Roles** tab.
+
+5. Assign the custom Tenant role to a user or group to validate that the permissions are applied correctly.
+
+## Edit a Custom Role
```

Comment:
```suggestion
## Custom Role Edit
```
I feel like it is good to avoid having two headings that are completely the same because they create messy anchors